### PR TITLE
adds binary, adds transpilation + prepublish script, improved path handling

### PR DESCRIPTION
**Pull request by flipace**
This builds upon #8 and also adds the babel transpilation step to ensure backwards-compatibility with node v6.10+. It also improves path handling and preps the package.json for publishing on npm (by ensuring to run ```build``` command before publishing and on installation).

### CODE REVIEW

**Comment by chrisachard** [comment]
Hm, I get the error:

src/attributes.js: Invalid options type for foreign

Some googling looks like it could be a babel core version that's out of date or something?

**Comment by flipace** [comment]
hm, could you try to remove your node_modules and npm i again?

**Comment by chrisachard** [comment]
I did :/

**Comment by flipace** [comment]
i accidentaly removed babel-cli so it probably used some globally installed version :man_facepalming:

if it still doesn't work for you, i'll have to dig deeper 🕳

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "presets": [
+    [
+      "env",
+      {
+        "node": "6.10.0"
```
[the remark this answers is not in the document]
Comment (flipace) [reply .babelrc]: this just ensures that the transpiled code is compatible with node 6.10 upwards, so it's still compatible with node 8 but also with >=6.10

[the remark this answers is not in the document]
Comment (sibelius) [reply .babelrc]: node 8 will be LTS on october 31st, so close

**Comment by chrisachard** [comment]
@flipace That worked!  But when doing the actual conversion, python couldn't find scripts.label_image if you do the sketch-to-react-native command outside of the project directory

**Comment by flipace** [comment]
@chrisachard true 👍 i had to change the paths in the neural net too - they were hardcoded there, i changed it so that ```tempDir``` gets passed down to the ```childIsBadElement``` function.

works fine for me now, even outside of the project dir :)

bildschirmfoto 2017-10-19 um 22 23 52

**Comment by chrisachard** [comment]
woo! works.  Thanks @flipace !

**Comment by chrisachard** [comment]
@flipace Someone in the slack channel just got the error: cannot find module '../build'; is that the error that we got but fixed?  Do you remember the fix, or: do you know the fix?

**Pull request merged**